app/src/main/res/layout/activity_register.xml

```diff
@@ -0,0 +1,195 @@
+        android:textAlignment="center"
+        android:textColor="@color/black"
+        android:textSize="16sp"
+        android:layout_below="@id/tv_register_account"/>
+    
+    <EditText
+        android:id="@+id/register_name"
+        android:layout_width="362dp"
+        android:layout_height="42dp"
+        android:layout_marginStart="25dp"
+        android:paddingTop="11dp"
+        android:paddingStart="16dp"
+        tools:text="Julia Shakira P"
+        android:textSize="14sp"
+        android:textColor="#66000000"
+        android:background="@drawable/register_border"
+        android:layout_below="@id/tv_register_name_title"/>
+
+    <TextView
+        android:id="@+id/tv_register_email_title"
+        android:layout_width="wrap_content"
+        android:layout_height="24dp"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="8dp"
+        android:text="@string/email_address"
+        android:textAlignment="center"
+        android:textColor="@color/black"
+        android:textSize="16sp"
+        android:layout_below="@id/register_name"/>
+
+    <EditText
+        android:id="@+id/register_email"
+        android:layout_width="362dp"
+        android:layout_height="42dp"
+        android:layout_marginStart="25dp"
+        android:paddingTop="11dp"
+        android:paddingStart="16dp"
+        tools:text="youremail@domain.com"
+        android:textSize="14sp"
+        android:textColor="#66000000"
+        android:background="@drawable/register_border"
+        android:layout_below="@id/tv_register_email_title"/>
+
+    <TextView
+        android:id="@+id/tv_register_password_title"
+        android:layout_width="wrap_content"
+        android:layout_height="24dp"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="8dp"
+        android:text="@string/password"
+        android:textAlignment="center"
+        android:textColor="@color/black"
+        android:textSize="16sp"
+        android:layout_below="@id/register_email"/>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/register_password_layout"
+        android:layout_width="362dp"
+        android:layout_height="42dp"
+        android:layout_marginStart="25dp"
+        android:layout_below="@id/tv_register_password_title"
+        app:passwordToggleEnabled="true" >
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/register_password"
+            android:inputType="textPassword"
+            android:layout_width="362dp"
+            android:layout_height="42dp"
+            android:paddingTop="11dp"
+            android:paddingStart="16dp"
+            tools:text="testingpassword"
+            android:textSize="14sp"
+            android:textColor="#66000000"
+            android:background="@drawable/register_border" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+
+<!--    <TextView-->
```

Comment:
delete this first ya, if you want to comment this block of codes please add some information why you comment it or add some todo ya